### PR TITLE
Added the FAQ content, with a link to the markdown file

### LINKED ISSUE #31: Add a link to a markdown version of FAQ
relates to #24 

This should write the content for the faq, and linked to on the website as a markdown file (similar to the code of conduct). And migrated to #30 after the applications close 😄

### PR DESCRIPTION
**Pull request by torch2424**
closes #31 
relates to #24 

This adds a FAQ document to the repo (Adding the content), and adds a link to this document from the website 😄 

Screen Shot 2019-12-09 at 8 17 21 AM
Screen Shot 2019-12-09 at 8 17 29 AM
Screen Shot 2019-12-09 at 8 17 35 AM

### CODE REVIEW

**Comment by pranjal9599** [comment]
We should also link to channel it'll live streamed and later uploaded to

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Frequently Asked Questions
+
+## Is this a community event?
+
+Yes, the summit is organised by individual people in the WebAssembly communitty, and at the time of writing sponsored by Google and Mozilla.
```
Comment (ttraenkler) [inline FAQ.md]: community

File: pages/index.tsx [hunk 2]
```diff
@@ -52,9 +52,10 @@ const LandingPage: FC = () => (
     </OnelinerCentered>
 
     <Footer>
-      <a href="mailto:wasm-summit-2020@chromium.org">wasm-summit-2020@chromium.org</a>
-      <a href="https://github.com/WebAssemblySummit/webassemblysummit.github.io/blob/dev/CODE_OF_CONDUCT.md">Code of conduct</a>
-      <a href="https://twitter.com/search?q=%23WasmSummit&src=typed_query">#WasmSummit</a>
+      <NoBreak><a href="mailto:wasm-summit-2020@chromium.org">wasm-summit-2020@chromium.org</a></NoBreak>
```
Comment (ttraenkler) [inline pages/index.tsx]: on mobile the layout breaks, maybe because of this?

**Comment by ttraenkler** [comment]
@torch2424 I had time to checkout into your branch realized the problem has been there before and only affects screen widths narrower than a typical smartphone, so it's a none issue.

**Pull request merged**